core/modules/wmgr/python/client.py

```diff
@@ -100,7 +102,7 @@ def _processData(self, items):
             currentBuf.write(self._makeHeader(name, filename, content, headers))
 
             if name is not None:
-                if isinstance(data, types.StringTypes):
+                if isinstance(data, str):
```

Comment:
See comment below, although it only makes a difference if you are using python 2 `unicode` I think.